Documentation/ApiOverview/Logging/Logger/Index.rst

```diff
@@ -31,7 +31,7 @@ Use the :code:`LoggerAwareTrait` in your class to automatically instantiate :cod
    }
 
 
-Or, you can instantiate the Logger with :code:`makeInstance`.
+Or, you can instantiate the Logger with :code:`makeInstance`. Be aware that a simple Logger configuration must be made also.
```

Comment:
That's actually not the case. You can retrieve a logger for anything without further configuration. By default the messages will end up in the default TYPO3 log this way. See `LogManager::getConfigurationForLogger()` which will traverse and at least use `$GLOBALS['TYPO3_CONF_VARS]['LOG']['writerConfiguration']`.